src/utils/network/session-core.js

```diff
@@ -383,35 +383,37 @@ export const createDocumentContainer = async (session, podUrl) => {
 };
 
 /*
-  User Inbox Section
-
-  Functions here deal primarily with user inbox/outbox on PASS
+  User Message Section
+  
+  Functions here deal primarily with user messages on PASS
 */
 
 /**
  * Function that gets list of inbox TTL file messages and returns the messages as
  * JSON
  *
  * @memberof utils
- * @function getInboxMessageTTL
+ * @function getMessageTTL
  * @param {Session} session - Solid's Session Object {@link Session}
- * @param {inboxListObject[]} inboxList - List of inbox messages
- * @returns {Promise<inboxListObject[]>} inboxList - An array of inbox messages
+ * @param {string} boxType - The message box being called "Inbox" or "Outbox"
+ * @param {messageListObject[]} listMessages - List of messages
+ * @returns {Promise<messageListObject[]>} inboxList - An array of inbox messages
  * from the user's inbox on Solid in JSON format
  */
-export const getInboxMessageTTL = async (session, inboxList) => {
-  const inboxContainerUrl = getContainerUrl(session, 'Inbox', 'self-fetch');
+
+export const getMessageTTL = async (session, boxType, listMessages) => {
```

Comment:
There's a lot going on in this function. Is it possible to break up into smaller pieces? We could split the message fetching and message parsing into 2 functions at least. There are probably a few other simplifications that can be made as well.

Reply:
Have moved the parsing of the message TTL file into session-helper as the new parseMessageTTL function.